spec-tests/src/main/java/org/candlepin/spec/bootstrap/assertions/JobStatusAssert.java

```diff
@@ -36,6 +37,68 @@ public static JobStatusAssert assertThatJob(AsyncJobStatusDTO actual) {
         return new JobStatusAssert(actual);
     }
 
+    /**
+     * Asserts the job completes within a reasonable amount of time, waiting if necessary. This
+     * assertion will fail if the job does not terminate in the specified duration from the time
+     * this assertion is invoked. A job is considered "terminated" when it enters a terminal state
+     * (i.e. FINISHED, FAILED, CANCELED, etc.).
+     * <p>
+     * The object under test will be mapped to the final AsyncJobStatusDTO upon successful
+     * completion.
+     *
+     * @param client
+     *  the client to use to verify the job has completed
+     *
+     * @return
+     *  a new JobStatusAssert for the completed job status
+     */
+    public JobStatusAssert terminatesWithinDuration(ApiClient client, long duration) {
```

Comment:
Missing the duration param in the javadocs

Reply:
Fixed